assets/telemeter-client/kube-rbac-proxy-secret.yaml

```diff
@@ -15,4 +15,9 @@ stringData:
         "user":
           "name": "system:serviceaccount:openshift-monitoring:prometheus-k8s"
         "verb": "get"
+      - "path": "/federate"
```

Comment:
hmm this should go to `assets/prometheus-k8s/kube-rbac-proxy-secret.yaml` rather as this is the configuration of the prometheus-k8s's kube-rbac-proxy sidecar that needs to allow /federate?
Conversely the telemeter-client service account needs the GET permission on /federate.